### ISSUE TITLE
Repeater's input suffix/prefix action

### ISSUE THREAD

**Issue by invaders-xx** [filamentphp/filament#3152]
### Package

filament/filament

### Package Version

v2.14.0

### Laravel Version

v9.20.0

### Livewire Version

v2.10.6

### PHP Version

PHP 8.1

### Bug description

I have an input with a suffix action : 
```php
 Forms\Components\TextInput::make('batch_number')
                                        ->label(__('Batch Number'))
                                        ->default(null)
                                        ->lazy()
                                        ->tap(function (Forms\Components\TextInput $component)  {
                                            $search_action = Forms\Components\Actions\Action::make('search_batch_number')
                                                ->tooltip(__('Search'))
                                                ->icon('heroicon-o-search')
                                                ->action(function (Component $livewire) use ($component, $record) {
                                                    $batch_number = $component->getState('batch_number');
                                                    ray($batch_number);
                                            $component->registerActions(['search_batch_number' => $search_action]);
                                            $component->prefixAction($search_action);
                                                });
``` 
It works in a Card/Grid component but not in a Repeater. I can see there is a livewire roundtrip but nothing happens. The $batch_number is not displayed in ray.

**Comment by invaders-xx** [comment]
@zepfietje, #3149 does not fix it

**Comment by invaders-xx** [comment]
@zepfietje, you can clone https://github.com/invaders-xx/filamentdemo.git, in product resource you have 2 field called "batch number", the first one is in a grid (the ray output works fine) and second one is in a repeater (the ray output is never reached)

**Comment by danharrin** [comment]
Fixed by #3413.
